### PR TITLE
CXINT-472: S/4 HANA Synchronous Order Management

### PR DESCRIPTION
**Pull request by SherwinVarghese**
**Note: Analogical PR in Spartacus repo's branch: #16650**

----

- S/4 HANA Synchronous Order Management
- Added changes for displaying the estimated delivery schedules for products in cart. The `scheduleLines` are fetched from the ERP system and provided in the OCC API.
- Schedule lines are visible in the Cart page, Order details page and Add to Cart pop up.

Cart Page
CART-Entry

Order Details Page
ORDER-Entry

### CODE REVIEW

File: integration-libs/s4om/root/components/schedule-lines/schedule-lines.component.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Component, Optional } from '@angular/core';
```
Comment (szczepaniak-michal) [inline integration-libs/s4om/root/components/schedule-lines/schedule-lines.component.ts:7]: Let's add E2E tests to check scheduline lines

**Review by szczepaniak-michal — CHANGES_REQUESTED** [review]
E2Es

File: integration-libs/s4om/root/components/schedule-lines/schedule-lines.component.ts (hunk 1 above)
Comment (SherwinVarghese) [reply integration-libs/s4om/root/components/schedule-lines/schedule-lines.component.ts:7]: done @szczepaniak-michal

**Comment by SherwinVarghese** [comment]
E2Es Passing

Screenshot 2022-12-08 at 12 55 50 AM

File: integration-libs/s4om/.release-it.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (RadhepS) [inline integration-libs/s4om/.release-it.json]: This file can be deleted since its no longer needed due to releasing using RBSC

Comment (SherwinVarghese) [reply integration-libs/s4om/.release-it.json]: noted. thanks @RadhepS

**Review by szczepaniak-michal — DISMISSED** [review]

**Comment by RadhepS** [comment]
Please ping me when this gets merged so I can update the release files to make sure that this is released as well @SherwinVarghese

**Comment by Platonn** [comment]
Closing in favor of the PR https://github.com/SAP/spartacus/pull/16650, to allow collaboration of core team people

**Comment by Platonn** [comment]
Reopening the PR, because the PR link is important for @SherwinVarghese's JIRA workflow. And we got permissions to Sherwin's repo.

**Comment by SherwinVarghese** [comment]
thanks @Platonn and @szczepaniak-michal

**Comment by SherwinVarghese** [comment]
Closing this PR as it is a duplicate of [16600](https://github.com/SAP/spartacus/pull/16600/)

**Pull request closed without merging**